Integrations/src/main/java/io/deephaven/integrations/learn/Output.java

```diff
@@ -28,13 +30,28 @@ public Output(String colName, PyObject scatterFunc, String type) {
         this(colName, new PythonFunctionCaller(Require.neqNull(scatterFunc, "scatterFunc")), type);
     }
 
-    private Output(String colName, Function<Object[], Object> scatterFunc, String type) {
+    /**
+     * Creates a new Output.
+     *
+     * @param colName       name of new column to store results.
+     * @param scatterFunc   function to scatter the results of a Python object into the table column.
+     * @param type          desired datatype of the new column.
+     */
+    public Output(String colName, Function<Object[], Object> scatterFunc, String type) {
 
         Require.neqNull(colName, "colName");
         Require.neqNull(scatterFunc, "scatterFunc");
 
         NameValidator.validateColumnName(colName);
 
+        List<String> validTypes = Arrays.asList("int", "long", "double", "float", "boolean", "char", "String");
+        if (type != null) {
+            if (validTypes.contains(type));
+            else {
+                throw new IllegalArgumentException("Output column datatype must be one of: int, long, double, float, boolean, char, String.");
+            }
+        }
```

Comment:
So does this mean that passing an invalid datatype like `"hello"` should just throw whatever it's going to throw when it gets there? This seems like a bad idea because all of the actual computation has to happen before the error is caught.